docs/src/main/asciidoc/http-reference.adoc

```diff
@@ -231,8 +231,12 @@
~`redirect`:: HTTP requests will be redirected to the HTTPS port
~`disabled`:: The HTTP port will not be opened.
 
 NOTE: if you use `redirect` or `disabled` and have not added an SSL certificate or keystore, your server will not start!
 
+=== Advanced SSL configuration
```

Comment:
To be honest, I don't think this section is necessary

Reply:
Okay, I have removed this part. Thank you!